docs/intro/narrative.rst

```diff
@@ -0,0 +1,133 @@
+workflow in various ways, and in this book you will use many of them right
+from the start.
+There are many topics you can explore, if you wish:
+Local or collaborative workflows, reproducible analyses, data publishing, ... .
+If anything seems particulary exciting, you can go ahead, read it, *and do it*.
+Therefore, **grab your computer, and be ready to use it**.
+
+Every chapter will give you different challenges, starting from basic local
+workflows to more advanced commands, and you will see your skills increase
+with each. While learning, it will be easy to
+**find use cases in your own work for the commands you come across**.
+
+As the handbook is to be a practical guide it includes as many hands-on examples
+as we can fit into it. Code snippets look like this, and you should
+**copy them into your own terminal to try them out**, but you can also
+**modify them to fit your custom needs in your own use cases**.
+Note in the example below that shows the creation of a DataLad dataset how
+we distinguish ``comments (#)`` from ``commands ($)`` and their output:
+
+.. code-block:: bash
+
+   # this is a comment used for additional explanations. Anything preceded by $ is a command to try.
+   # if the line starts with neither # nor $, its the output of a command
+   $ datalad create myfirstrepo
+   [INFO   ] Creating a new annex repo at /home/adina/DataLad-101
+   create(ok): /home/adina/DataLad-101 (dataset)
+
+The following two chapters have two different goals: All of the *Basics* sections
+will show you the core DataLad functionality and challenge you to use it.
+In *use cases* you will find concrete examples of DataLad applications for
+general inspiration. We recommend to read the Basics first. Afterwards,
+you might even consider :ref:`contribute` to this book by sharing your own use case.
+
+Note that many challenges can have straightforward and basic solutions,
+but a lot of additional options or improvements are possible.
+Sometimes one could get lost in all of the available DataLad functionality.
+For this reason we put all of the basics in plain sight, and those basics
+will let you master a given task and get along comfortably.
+Having the basics will be your multi-purpose swiss army knife.
+But if you want to have the special knowledge for a very peculiar type
+of problem set or that extra increase in skill,
+you'll have to do a detour into some of the *hidden* parts of the book:
+When there are command options that go beyond basics and
+best practices, we hide them in foldable book sections in order
+to not be too distracting for anyone only interested in the basics.
+You can decide for yourself whether you want to check them out:
+
+.. container:: toggle
+
+    .. container:: header
+
+       **Additional Content: Click here to show/hide further commands**
+
+    Sections like this contain content that goes beyond the basics
+    necessary to complete a challenge.
+
+Note further that...
+
+.. admonition:: Note for git-users
```

Comment:
See #44 for a related proposal

Reply:
thx! I'm looking forward to trying what you have implemented there